### PR TITLE
refactor(hubble): enhance maven front plugin

### PR DESCRIPTION
**Pull request by liuxiaocs7**
## Purpose of the PR

follow: #558

## Verifying these changes

- [x] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [ ] Need tests and can be verified as follows:
    - xxx

## Does this PR potentially affect the following parts?

- [x]  Nope
- [ ]  Dependencies (add/update license info) 
- [ ]  Modify configurations
- [ ]  The public API
- [ ]  Other affects (typed here)

## Documentation Status

- [ ]  `Doc - TODO` 
- [ ]  `Doc - Done` 
- [x]  `Doc - No Need`

### CODE REVIEW

**Comment by liuxiaocs7** [comment]
cc @imbajin and @aroundabout

maybe need to test when build image

File: hugegraph-hubble/hubble-dist/pom.xml [hunk 1]
```diff
@@ -73,21 +76,24 @@
...
                         <configuration>
-                            <nodeVersion>v16.16.0</nodeVersion>
-                            <npmVersion>8.19.1</npmVersion>
-                            <yarnVersion>v1.22.15</yarnVersion>
+                            <nodeVersion>${build.node.version}</nodeVersion>
+                            <npmVersion>${build.npm.version}</npmVersion>
+                            <yarnVersion>${build.yarn.version}</yarnVersion>
+                            <nodeDownloadRoot>https://npm.taobao.org/mirrors/node/</nodeDownloadRoot>
+                            <npmDownloadRoot>https://registry.npmmirror.com/npm/-/</npmDownloadRoot>
+                            <yarnDownloadRoot>https://repo.huaweicloud.com/yarn/</yarnDownloadRoot>
```
Comment (imbajin) [inline hugegraph-hubble/hubble-dist/pom.xml:95]: could we use the same mirror url from taobao?

Comment (liuxiaocs7) [reply hugegraph-hubble/hubble-dist/pom.xml]: Done~

Comment (liuxiaocs7) [reply hugegraph-hubble/hubble-dist/pom.xml:92]: seems not now, The download address is automatically spliced based on url and version

```
https://repo.huaweicloud.com/yarn/v1.22.21/yarn-v1.22.21.tar.gz

https://registry.npmmirror.com/yarn/-/yarn-1.22.21.tgz
```

but we can not download file from this url below.

```
https://registry.npmmirror.com/yarn/v1.22.21/yarn-v1.22.21.tar.gz
```

File: hugegraph-hubble/hubble-dist/pom.xml [hunk 2]
```diff
@@ -73,21 +75,22 @@
...
-                            <goal>install-node-and-npm</goal>
                         </goals>
                         <configuration>
-                            <nodeVersion>v16.16.0</nodeVersion>
-                            <npmVersion>8.19.1</npmVersion>
-                            <yarnVersion>v1.22.15</yarnVersion>
+                            <nodeVersion>${build.node.version}</nodeVersion>
+                            <yarnVersion>${build.yarn.version}</yarnVersion>
+                            <nodeDownloadRoot>https://mirrors.aliyun.com/nodejs-release/</nodeDownloadRoot>
+                            <yarnDownloadRoot>https://repo.huaweicloud.com/yarn/</yarnDownloadRoot>
```
Comment (imbajin) [inline hugegraph-hubble/hubble-dist/pom.xml:92]: Skip it now & replace it(the mirror address) in future

**Review by imbajin — APPROVED** [review]

**Review by javeme — APPROVED** [review]

**Comment by imbajin** [comment]
related issue #584

**Pull request merged**